src/main.rs

```diff
@@ -29,7 +29,7 @@ struct Args {
 
     #[clap(
         long,
-        default_value = "app.kubernetes.io/managed-by=linkerd-failover",
+        default_value = "failover.linkerd.io/controlled-by=linkerd-failover",
```

Comment:
```suggestion
        default_value = "failover.linkerd.io/controlled-by",
```

We could just select on the label's existence by default...

Reply:
Ah I like that!